base/libgit2/tree.jl

```diff
@@ -16,19 +16,22 @@ function treewalk(f::Function, tree::GitTree, payload=Any[], post::Bool = false)
     return cbf_payload
 end
 
-"Get the filename of a tree entry."
+"""Returns a file name, as `String`, of a tree entry.
+
+In case of error, `nothing` is returned.
```

Comment:
What kind of errors are possible? Ignoring them and returning `nothing` is an odd strategy for error handling.